### PR TITLE
Develop fix callback

### PR DESCRIPTION
**Pull request by montazarr**

### CODE REVIEW

File: includes/dintero-checkout-functions.php [hunk 1]
```diff
@@ -103,10 +103,6 @@ function dintero_update_wc_shipping( $data ) {
  * @return void
  */
 function dintero_confirm_order( $order ) {
-	if ( ! empty( $order->get_date_paid() ) ) {
```
Comment (MichaelBengtsson) [inline includes/dintero-checkout-functions.php:106]: Why is this removed @montazarr. What would happen if a customer triggers this function multiple times?

Comment (montazarr) [reply includes/dintero-checkout-functions.php:106]: The date paid is only set when the order is processing or completed. Since orders that are created on callback is pending payment, they don't have a date paid, which prevents the order from being processed. This, I suggest we have to remove this check.

I don't think the would have issue with orders being processed multiple times since we already have checks for that in the order management class.

Comment (MichaelBengtsson) [reply includes/dintero-checkout-functions.php:106]: Ok in that case all good!
